apps/view/src/render/models.ts

```
@@ -77,19 +77,27 @@ export async function boardToStackups(
     return Promise.all([selfContainedStackup, sharedStackup])
   })
 }
-
-export async function stackupToZipBlob(stackup: Stackup): Promise<Blob> {
+  export async function stackupToZipBlob(
```

ESLint

[error] Delete `··`

(prettier/prettier)